ssl/s3_lib.c

```diff
@@ -834,6 +834,21 @@ static SSL_CIPHER ssl3_ciphers[] = {
      256,
      256,
      },
+    {
```

Comment:
I'm not sure. We don't do that at the moment for any of the TLSv1.2 specific ciphers. I'd rather not litter the code with lots of ifdef if we don't have to, and it won't break if you compile without TLSv1.3 (and you won't be able to use these ciphers)....? Will do if you really want it though.